introduction/architecture.md

```diff
@@ -111,6 +110,64 @@ while maintaining support for multiple indexes.
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store regular, non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes are not
+aware of cluster-wide distribution policies, and otherwise look just as if they were single TimescaleDB
+instances. However, certain actions are blocked on data nodes to protect from users accidentally
+accessing the data nodes directly.
+
+It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
+purposes, look just like a single node of TimescaleDB from an operational perspective.
+
+# Distributed Databases
+
+Distributed databases represent the highest level in the hierarchy used to define a multi-node
+TimescaleDB Cluster. Users can define access nodes and data nodes at the database level. This means
+that if you have multiple databases in a TimescaleDB Cluster, you could theoretically have
+different nodes play both the role of access node and data node for different databases.
+We do not typically suggest running in this manner. Instead, we typically recommend that you
+only have one front-end node that acts as a query routing layer between your application and
+all of your backend server nodes.
+
+# Distributed Hypertables
+
+As discussed above, distributed databases contain distributed hypertables. Distributed hypertables are essentially
+the same as regular hypertables, except that they have an additional layer of abstraction where
+access nodes must be able to logically manage distributed hypertables. Distributed hypertables
+are also composed of chunks.
+
+With distributed hypertables, it is suggested to partition the hypertables both by space and by time.
+If you only partition data by time, that chunk will have to fill up before the access node chooses another
+data node to store the next chunk. On the other hand, if you specify a space partition, the access node
+will distribute chunks across multiple data nodes based on the space partition so that multiple
+chunks are created for a given chunk time interval.
+
+# Scaling out with TimescaleDB Clustering
+
+As time-series data grows, a common use case is to add data nodes to expand the storage and compute
+capacity of your TimescaleDB Cluster. TimescaleDB can be scaled out by simply adding data nodes to
+a distributed database.
+
+There are certain cases where you may want to change your partitioning scheme. Because TimescaleDB is
+built specifically for time-series, we leverage the concept of time to provide more flexibility when
+changing partitions. Unlike many existing NoSQL databases where partitioning settings cannot be
```

Comment:
i think thats only true for time time dimension but not for space dimensions

Reply:
hmmmmm I think you can change the space dimension, and newer chunks will respect that partitioning scheme? cc @erimatnor

Reply:
Ok, spoke with @svenklemm offline and apparently this is fine, so going to go ahead and resolve this.